### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by shyusu4**
## PR Description

- Applied React best practices and language style guides in code.
- Final design matches Zeplin's design.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,39 +1,205 @@
...
-  text-align: center;
+* {
+  margin: 0;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+body {
+  margin: 0;
```
Comment (tufoinnkuo10) [inline src/App.css:6]: - [ ] Wonderful job adding the random percentages. please kindly add a margin bottom like in the zeplin design. i can barely see where to add a new book . great job
your design
Screenshot (490)_LI

zeplin
Screenshot (489)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
## Hi @shyusu4 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅Design well followed

## Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/App.css (hunk 1 above)
Comment (shyusu4) [reply src/App.css:6]: Hi, thanks for the review! I'll add the margin-bottom. 👍

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @shyusu4     , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 
- No linter errors 
- Nice UI design


Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**